projects/ui/src/Components/Home/HomePage.style.tsx

```diff
@@ -87,10 +87,9 @@ export namespace HomePageStyles {
       // This is a best-guess background fit attempt for user-supplied images.
       height: 100%;
       ${mediaQueryWithScreenSize.mediumAndSmaller} {
-        height: unset;
         width: 100%;
       }
-      background-size: cover;
+      object-fit: cover;
```

Comment:
Very nice! I actually used this once on a personal project once ages ago, but I 1000% forgot this existed; thanks for the reminder ;)